src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewClassificationAttributes.java

```diff
@@ -67,10 +80,17 @@
      */
     public ViewClassificationAttributes(final Composite parent,
                                     final Controller controller) {
+        
+        // Load images
+        IMAGE_INSENSITIVE       = controller.getResources().getManagedImage("bullet_green.png"); //$NON-NLS-1$
+        IMAGE_SENSITIVE         = controller.getResources().getManagedImage("bullet_purple.png"); //$NON-NLS-1$
+        IMAGE_QUASI_IDENTIFYING = controller.getResources().getManagedImage("bullet_yellow.png"); //$NON-NLS-1$
+        IMAGE_IDENTIFYING       = controller.getResources().getManagedImage("bullet_red.png"); //$NON-NLS-1$

```

Comment:
Moreover, we could add specific icons for the class attribute, e.g. by extending the method to getImage(AttributeType t, boolean isClassAttribute). If the flag is set to true, an icon could be returned which, e.g., contains a black outline around the dots.